internal/databases/sqlserver/blob/server.go

```diff
@@ -624,12 +624,15 @@ func (bs *Server) HandleBlobPut(w http.ResponseWriter, req *http.Request) {
 			return
 		}
 	}
+	bs.indexesMutex.Lock()
+	defer bs.indexesMutex.Unlock()
```

Comment:
It is seems that we can get this lock after `idx.Save` execution.

Reply:
We would like to be more conservative here.
And prevent succesful GET requests before PUT accomplished.